client/post-editor/editor-drawer/index.jsx

```diff
@@ -261,6 +262,17 @@ const EditorDrawer = React.createClass( {
 		);
 	},
 
+	renderCopyPost: function() {
+		// TODO: replace with currentPostTypeSupports( 'copy' );
```

Comment:
Alright, I'll remove that comment, as it's now clear to me that it doesn't make much sense.

@aduth I don't think there's a reason. When I started I already considered pages too, but then I decided to defer it to a future dedicated branch, to prevent this branch to grow indefinitely.